annotationPipeline/src/main/resources/application.properties.EXAMPLE

```diff
@@ -2,5 +2,5 @@ spring.batch.job.enabled=false
 genomenexus.hgvs=<HOST>/variant_annotation/hgvs/
 genomenexus.xrefs=<HOST>/xrefs/
 genomenexus.isoform_query_parameter=isoformOverrideSource
-genomenexus.hotspot_parameter=summary
+genomenexus.enrichment_fields=fields=hotspots,mutation_assessor
```

Comment:
@angelicaochoa Sorry, this isn't entirely what I meant. Given that the prop is called `enrichment_fields`, it seems redundant to include `fields=` in the value. Better to update the code from `"&" + enrichmentFields` to `"&fields=" + enrichmentFields`. Not that important tho